packages/components/ng-at-internet/src/constants.js

```diff
@@ -23,44 +29,59 @@ export const AT_INTERNET_CUSTOM_VARS = {
    */
 
   referrerSite: {
-    path: 'site.8', // OVH's AtInternet configuration
+    path: {
+      default: 'site.8', // OVH's AtInternet configuration
+      US: 'site.11',
+    },
     format: '[%s]',
   },
 
   /**
    * Event
    */
   event: {
-    path: 'site.20',
+    path: {
+      default: 'site.20',
+      US: 'site.6',
+    },
     format: '[%s]',
   },
 
   siteName: {
-    path: 'site.13',
+    path: {
+      default: 'site.13',
+      US: 'site.14',
+    },
     format: '[%s]',
   },
 
   /**
    * Project Id
    */
   projectId: {
-    path: 'page.1', // OVH's AtInternet configuration
+    path: {
+      default: 'page.1', // OVH's AtInternet configuration
```

Comment:
This doesn't exist for US or it is the same ?

Reply:
Yep, not specified in MANAGER-6432